src/Models/CardMarket.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace Pokemon\Models;
+
+/**
+ * Class CardMarket
+ *
+ * @package Pokemon\Models
+ */
+class CardMarket extends Model
+{
+
+    /**
+     * @var string
+     */
+    private $url;
+
+    /**
+     * @var string|null
+     */
+    private $updatedAt;
+
+    /**
+     * @var CardMarketPrices|null
+     */
+    private $cardMarketPrices;
```

Comment:
This property name needs to match it's getter and setter name, or `toArray()` and `toJson()` fails.
```suggestion
    private $prices;
```